app/components/mandatarissen/mandataris/publication-status-selector.js

```diff
@@ -51,6 +55,13 @@ export default class MandatarissenMandatarisPublicationStatusSelectorComponent e
       this.isDisabled = true;
     }
     this.mandataris.publicationStatus = publicationStatus;
+    if (publicationStatus?.uri === MANDATARIS_EFFECTIEF_PUBLICATION_STATE) {
+      this.args.mandataris.effectiefAt = new Date();
+    }
+    if (publicationStatus?.uri === MANDATARIS_DRAFT_PUBLICATION_STATE) {
+      this.args.mandataris.effectiefAt = null;
+    }
```

Comment:
This is done so when switching to bekrachtigd status this value is not removed, onlt when setting back to draft status.